mkdocs.yml

```
@@ -0,0 +1,38 @@
+  name: mkdocs
+  features:
+    - navigation.tabs
+    - navigation.sections
+    - content.code.annotate
+markdown_extensions:
+  - admonition
+  - codehilite
+  - toc:
+      permalink: true
+extra_css:
+- css/mkdocstrings.css
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 0

(indentation)